README.md

```diff
@@ -131,4 +143,53 @@ func MyHandler(w http.ResponseWriter, r *http.Request) {
 }
 ```
 
-Note: This is not required, but is highly recommended. By setting a customer ID you can easily associate requests with your customers/users in the Speakeasy Dashboard, powering filters in the Request Viewer [(Coming soon)](https://docs.speakeasyapi.dev/speakeasy-user-guide/request-viewer-coming-soon).
\ No newline at end of file
+Note: This is not required, but is highly recommended. By setting a customer ID you can easily associate requests with your customers/users in the Speakeasy Dashboard, powering filters in the Request Viewer [(Coming soon)](https://docs.speakeasyapi.dev/speakeasy-user-guide/request-viewer-coming-soon).
```

Comment:
I updated the docs, if you want to change the `(Coming soon)` to something else

Reply:
done